.github/workflows/go.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow will build a golang project
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-go
+
+name: Go
+
+on: push
+
+jobs:
+
+  build:
```

Comment:
Nit-pick but this job name implies build only but analysis and test are included. Suggest changing job name to reflect activities or split out into multiple sequential jobs.

Reply:
Good catch! I modified one of the templates and missed this. I'll look into your suggestions and see what aligns with the Gitlab Actions best practices.